pom.xml

```
@@ -58,12 +58,12 @@
     <maven.compiler.proc>full</maven.compiler.proc>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.build.outputTimestamp>2025-04-03T13:23:00Z</project.build.outputTimestamp>
-    <cloud-sdk.version>5.18.0</cloud-sdk.version>
-    <junit-jupiter.version>5.12.2</junit-jupiter.version>
+    <cloud-sdk.version>5.19.0</cloud-sdk.version>
```

Revert Cloud SDK update
```suggestion
    <cloud-sdk.version>5.18.0</cloud-sdk.version>
```